scripts/docs/extension_instructions.md

```diff
@@ -168,23 +105,9 @@ The recommended structure for Bareflank extensions is as follows:
    | - build/
    | - hypervisor/
    | - my_extension/ (<-- extension directory)
-       | - build_rules.cmake
        | - CMakeLists.txt
-       | - config.cmake
        | - include/
        | - src/
-       | - test/
+            | - CMakeLists.txt
+       | - tests/
```

Comment:
no need for this

Reply:
do you mean the whole section, or just the `tests` subfolder?

Reply:
tests